src/io.cpp

```diff
@@ -379,7 +381,7 @@ bool save_result_msh(const std::string& filename,
                      const std::vector<std::vector<size_t>>& shells,
                      const std::vector<std::vector<size_t>>& cells)
 {
-    constexpr size_t INVALID = std::numeric_limits<size_t>::max();
+    constexpr size_t INVALID = SIZE_MAX;
```

Comment:
Hmm, what error does `std::numeric_limits<size_t>::max()` cause?

Reply:
I was getting compiler errors, ideally the macro definition for the `size_t`, i.e. `SIZE_MAX` is the same as the function call `std::numeric_limits<size_t>::max()`.

See the related post on [stack overflow](https://stackoverflow.com/questions/40492414/why-does-stdnumeric-limitslong-longmax-fail).

Reply:
looks good to me.

Reply:
Updated!